_stbt/match.py

```diff
@@ -1013,6 +1036,7 @@ def link(name, level=None, match=None):  # pylint: disable=redefined-outer-name
         template,
         ConfirmMethod=ConfirmMethod,
         link=link,
+        fast_path=imglog.data.get("fast_path"),
```

Comment:
The keyword args were sorted alphabetically before.

Reply:
Fixed.